integration-tests/src/test/groovy/com/okta/sdk/tests/it/util/OktaOrgCleaner.groovy

```diff
@@ -60,27 +63,81 @@ class OktaOrgCleaner {
 
         log.info("Deleting Groups:")
         client.listGroups().stream()
-                .filter { it.getProfile().getName().matches(".*-${uuidRegex}.*")}
-                .forEach {
-                    log.info("\t ${it.getProfile().getName()}")
-                    it.delete()
-                }
+            .filter { it.getProfile().getName().matches(".*-${uuidRegex}.*") }
+            .forEach {
+                log.info("\t ${it.getProfile().getName()}")
+                it.delete()
+            }
 
         log.info("Deleting Group Rules:")
         client.listGroupRules().stream()
-                .filter { it.getName().matches("rule\\+${uuidRegex}.*")}
-                .forEach {
-                    GroupRule rule = it
-                    log.info("\t ${rule.getName()}")
-                    Util.ignoring(ResourceException) {
-                        rule.deactivate()
-                    }
-                    rule.delete()
+            .filter { it.getName().matches("rule\\+${uuidRegex}.*") }
+            .forEach {
+                GroupRule rule = it
+                log.info("\t ${rule.getName()}")
+                Util.ignoring(ResourceException) {
+                    rule.deactivate()
                 }
+                rule.delete()
+            }
 
         log.info("Deleting Policies:")
         client.listPolicies(PolicyType.OKTA_SIGN_ON.toString()).stream()
-            .filter { it.getName().matches("policy\\+${uuidRegex}.*")}
+            .filter { it.getName().matches("policy\\+${uuidRegex}.*") }
+            .forEach {
+                it.delete()
+            }
+
+        log.info("Deleting LinkedObjectDefinitions:")
+        client.listLinkedObjectDefinitions().stream()
+            .filter { it.getName().startsWith("manager") }
+            .forEach {
+                it.setName(it.getPrimary().getName())
+                it.delete()
+            }
+
+        log.info("Deleting InlineHooks:")
+        client.listInlineHooks().stream()
+            .filter { it.getName().matches(".*-${uuidRegex}.*") }
+            .forEach {
+                it.deactivate()
+                it.delete()
+            }
+
+        log.info("Deleting EventHooks:")
+        client.listEventHooks().stream()
+            .filter { it.getName().matches(".*-${uuidRegex}.*") }
+            .forEach {
+                it.deactivate()
+                it.delete()
+            }
+
+        log.info("Deleting UserTypes:")
+        client.listUserTypes().stream()
+            .filter { !it.getDefault() && it.getName().startsWith("java_sdk_user_type_") }
+            .forEach {
+                it.delete()
+            }
+
+        log.info("Deleting AuthorizationServers:")
+        client.listAuthorizationServers().stream()
+            .filter { !it.getName().equals("default") && it.getName().startsWith("java-sdk-authorization-server-") }
```

Comment:
nit: the previous example uses snake case, this one uses kabob case

My personal pref is to add the term `it` or `tests` in there, but this is only a suggestion

Reply:
addressed this.